tests/plugin.xml

```diff
@@ -29,5 +29,12 @@
     <js-module src="tests.js" name="tests">
     </js-module>
 
+    <platform name="ios">
+      <!-- The WKWebView implementation for inappbrowser requires the presence of this plugin  -->
+      <dependency id="cordova-plugin-wkwebview-engine" />
+      <dependency id="cordova-plugin-wkwebview-engine-allowfileaccess" url="https://github.com/knight9999/cordova-plugin-wkwebview-engine-allowfileaccess.git" />
```

Comment:
Without the code in that plugin, `cordova-paramedic` tests fail with WKWebView.
What do you suggest? Bundle the plugin code inside `tests/` within this plugin?

Reply:
The plugin itself is only 2 lines. There are a couple things we could do.

1. Ask @knight9999  to publish the allowfileaccess plugin to npm, and have our test depend on that published version.
2. make the wk-allowfileaccess a cordova supported plugin, and also do suggestion 1, but replace Ken with 'we'
3. revisit the decision to not allow file access directly in the wkwebview plugin, could we just default to NO, but allow the developer to turn it on? ( This would require a firm warning about app store rejection possibilities, so it might make things too easy for developers and lead to rejections. )
4. Include it in the paramedic repo as a spec plugin, as suggested by Ken [1]




[1] https://github.com/apache/cordova-paramedic/issues/52

Reply:
@knight9999 Ken, any preference in Jesse's above options? 
This isn't something I can resolve myself to unblock this PR: all of the above require action on the part of Cordova maintainers and it has wider implications in terms of test support for WKWebView on iOS, i.e. for `cordova-plugin-wkwebview-engine` and `cordova-ios@5`.

Reply:
I have published my plugin https://www.npmjs.com/package/cordova-plugin-wkwebview-engine-allowfileaccess .